frame/evm/src/lib.rs

```diff
@@ -1175,3 +1174,26 @@ impl<T> OnCreate<T> for Tuple {
 		)*)
 	}
 }
+
+/// Native system account provider that `frame_system` provides.
+pub struct NativeSystemAccountProvider<T>(core::marker::PhantomData<T>);
```

Comment:
Suggest: renaming to `FrameSystemAccountProvider`

Reply:
This is indeed more explicit, good suggestion!